### ISSUE TITLE
Add physicalInputDataSize to StatementStats

### ISSUE THREAD

**Issue by hashhar** [prestosql/presto#4553]
In https://github.com/prestosql/presto/pull/4354 `physicalInputDataSize` was added to `BasicQueryStats`. We should also add the same to `StatementStats` so that the query API response includes data scan stats.

Discussed at https://prestosql.slack.com/archives/CFPVDCDHV/p1594966585031100

**Comment by hashhar** [comment]
Some fields from `QueryInfo` which might be interesting for client libraries (can be targeted in future pull-requests):

1. `resourceWaitingTime`, `dispatchingTime`, `executionTime`, `analysisTime`, `planningTime` and `finishingTime` are not exposed. Can be useful to show in any interactive query tool (Redash/Superset etc.).
2. `peakUserMemoryReservation` is exposed  but `peakTotalMemoryReservation` is not. **Maybe this is intentional?**
3. `physicalInputDataSize` and `physicalInputPositions` are not exposed.
4. `outputDataSize` and `outputPositions` are not exposed.

**Comment by hashhar** [comment]
I'm confused about the scope of this change. Let me summarise my thoughts:

The initial goal was to expose `physicalInputDataSize` in StatementStats so that `v1/executing` resource can send it back to clients. This is done in https://github.com/prestosql/presto/pull/4600 (tests and some investigations pending).

Now when reading through the related code where the existing fields are used I observed the following:

1. The CLI uses `StatementStats.getProcessedBytes()` to show data as `0.45 [2.37M rows, 22.2MB] [5.24M rows/s, 49.1MB/s]`
2. The Web UI uses the `/v1/query` resource which shows:
    i. Input Data = `QueryInfo.QueryStats.getProcessedInputDataSize()`
    ii. Physical Input Data = `QueryInfo.QueryStats.getPhysicalInputDataSize()`

This means that the data-size that CLI reports can be misleading when compared to Web UI (but it should not be an issue since processedInputDataSize >= physicalInputDataSize generally).

**TL; DR: Should the CLI's StatusPrinter be changed to use `StatementStats.physicalInputDataSize` instead of `StatementStats.processedBytes`? Is there benfit to doing so or should it be left alone? The change can be challenging because newer clients with older server versions will break if we introduce the change without adding a conditional statement regarding presence of the fields.**

cc: @electrum
